vertx-web-openapi/pom.xml

```diff
@@ -48,7 +48,8 @@
       <groupId>io.vertx</groupId>
       <artifactId>vertx-web-api-service</artifactId>
       <version>${project.version}</version>
-      <scope>optional</scope>
+      <scope>compile</scope>
```

Comment:
I think this needs to remain optional

Reply:
scope optional doesn't exist afaik, i set optional true at the line below

Reply:
ah ok :-)